tests/unit/containers/quiz/QuizQuestion.spec.jsx

```diff
@@ -0,0 +1,40 @@
+import QuizQuestion from '~/containers/quiz/quiz-question/Question.tsx'
+import { render, screen } from '@testing-library/react'
```

Comment:
also import from packages we need to move up

Reply:
Thanks, moved it up!